js/focuspoint/jquery.focusPoint.js

```diff
@@ -290,7 +299,7 @@
 
 				return this;
 			},
-
+			
```

Comment:
Seems like there is something wrong with your coding standards, but not anything big.